scripts/build.sh

```diff
@@ -22,7 +22,7 @@ fi
 if [[ "${TARGET_PLATFORM}" == win-* ]]; then
     conda install -y "nsis=3.01" -c conda-forge --override-channels
 fi
-pip install git+git://github.com/chrisburr/constructor@64ebd6d34f0f18684c76c0bebcfab41c38d55083#egg=constructor --force --no-deps
+pip install git+https://github.com/conda/constructor@3.3.1#egg=constructor --force --no-deps
```

Comment:
conda-forge/constructor-feedstock#61 was merged, so we can remove this in a couple hours.

```suggestion
```